src/artofillusion/polymesh/QuadMesh.java

```diff
@@ -1,12 +1,16 @@
-/* Copyright © 2007 by François Guillet
+/* 
+  Copyright © 2007 by François Guillet
+  Editions copyright © 2023 Petri Ihalainen
```

Comment:
Small point of language here:

"Editions" is probably not the word you want. Despite appearances, it is not another way of saying "Edits" or "Changes". "Additions" might be appropriate if you are only *adding* code, but not when changing what is already there.

Reply:
Feel free to correct it. :)